xWorkflows/Xworkflow1/go/kubecost-event/main.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Kubecost API endpoint URL (replace with your actual URL)
+const kubecostAPIURL = "http://kubecost-cost-analyzer.kubecost.svc.cluster.local:9003/allPersistentVolumes"
```

Comment:
Using env var, and providing variable values using env var in pod.spec yaml file

Reply:
added env vars for these values